pkg/kp/kv_test.go

```diff
@@ -53,6 +53,14 @@ func TestGetHealthWithEntry(t *testing.T) {
 }
 
 func makeStore(t *testing.T) (Store, *testutil.TestServer) {
+	// Detect if the test gets skipped by testutil.NewTestServerConfig.
+	// This happens if 'consul' isn't in $PATH
+	defer func() {
+		if t.Skipped() {
+			t.Fatal("test skipped by testutil package")
```

Comment:
After looking at the output generated by a skipped-then-failed run, something like "failing skipped test" would make more sense in context.